### ISSUE TITLE
A batch of proposals

### ISSUE THREAD

**Issue by IvanGBi** [lobster-dao/overview#10]
1] Of course, the first issue is what to do with the 49% of supply the DAO will end up with. I honestly don't know what is best here. I don't think burning all is correct, but maybe the majority of them? See twitter post by the [account](https://twitter.com/10b57e6da0/status/1446786579918467072).

2] Also, wanna see a DAO proposal for X number of NFTs for the following:
a) best recaps of solana-avalanche-fantom-polygon-xdai discussions why yes/no
b) recap of the topic on regulations and how to approach token-security (non-legal advice)
c) recap on malicious<>hack<>arbitrage<>theft semantic debate

3] And an issue for those [who were late or didn't know](https://twitter.com/10b57e6da0/status/1446786579918467072). The database didn't keep records of who did or did not claim, it's an anonymized set of addresses. So you can't trust someone saying if they did or did not claim, unless we ask the bot developer to look into it. But a few founders forgot to claim, and that's sad. What could be done is the following, imho:
a] ask bot developer to add more registrants and then give those people only 1 even if they had 5 before, as a fee.
b] collect only founders who can be trusted and just distribute to them, without asking the bot dev to check for sybil
c] ?

More incoming!

**Comment by alexflorence** [comment]
1) In my opinion the main issue here is that 49% of the supply are in danger of being controlled by a minority. In an ideal governance case we would have 51% of the supply voting for what happens with the 49% but the reality of governance looks different. It may very well be that in practical terms we end up with a very small share of the supply voting on what happens with almost half of the supply. So we have to get the 49% share down somehow.

I can think of 3 solutions to resolve this:

A) Burn the DAO treasury down to a reasonable share of the total supply. Not all people are happy with this since burning the art seems destructive.

B) Instead of burning the treasury down to a reasonable share the excess NFTs could be distributed proportionally to existing holders or the initial minters.

C) Only allow governance decisions to affect as many NFTs from the treasury has have been used to vote on the proposal. (This is the most simplified version of this model. It might be that changing it to 200%, or some other value, of the amount of NFTs have been voted with makes more sense.)

______

2) Do you mean retroactively for existing messages in the LobsterDAO or would this be some sort of content creation contest with the winners being awarded by the DAO?

______

3) This will be a never ending story because there are plenty of people who did not claim

A) Personally I think this is reasonable.
B) has the heavily subjective component of "founders who can be trusted" (no offense to anyone. I am taking a devils advocate POV here.)
C) The Telegram bot still responds to me saying that my Telegram ID is not on the claimers list. The question is whether it says that for any Telegram ID  now because the list was deleted  - or because the data is still available. If the data is still available I am favor of resolving the issue based on that because its the least subjective solution.

**Comment by IvanGBi** [comment]
<img width="783" alt="Screenshot 2021-10-09 at 13 48 11" src="https://user-images.githubusercontent.com/40795256/136656702-3a911347-c361-4013-b06f-e57a84ba75cc.png">

Ah and one more - this was my bad! The user didn't have tooooo many messages, but was valuable. I believe the number this would qualify for is 1. The DAO should gib the 1 piece, this is fairly trivial. Screenshot as proof.

**Comment by msirinotis** [comment]
For #3.. 3a) helps to solve #1 right? 
Diversify governance with existing eligible users/addresses and brings the 49% of the DAO down (and closer to the initial 83% community allocation). 
If people miss the second claim round then I think it's somewhat safe to assume they are not currently following the community and then can look at burn/distribute beyond that.

* Dependant on identifying those that went unclaimed.

**Comment by amplice** [comment]
I don't have strong feelings about what to do for people who missed it.

I think with regards to what to do with big amount of DAO owned ones, we should do something like 1 lobster to every gitcoin donator (try to remove sybil somehow). Or donate a big chunk for public goods funding of some kind.

LobsterDAO should be trying to encourage prosocial, non zero sum behavior in the community by rewarding those who have demonstrated this behavior in the past.

Maybe something like, rewarding this comp users who gave comp tokens back, rewarding previous white hat hackers who returned funds (esp if they did so immediately) - things like this.

Basically rewarding 'good' behavior across the board of all kinds + public goods funding in whatever way we can. This not only increase distribution, but also distributes to good actors.

**Comment by IvanGBi** [comment]
@arbitary you want to defeat the exact purpose of the convo: to appreciate ALL members who missed, not to enrich the current. Totally 180. Imho: bad.

**Comment by IvanGBi** [comment]
@arbitary touche. haha.

**Comment by Apestuff** [comment]
I would love for the following to happen: 

1) donation of few select pieces for historical and cultural preservation to established art institutions for display in virtual form. 

2) form a prize pool for significant community contribution in future, e.g. significant white hat discovery preventing loss of funds . i.e. badge of honour for keeping to the ethos of the community

**Comment by Fortuno89** [comment]
I agree with ivan’s proposal, both about the distribution of one nft to those founders who have not claimed and also the need to burn the NFTs which will remain on the DAO (70-30 sounds ok). It all sounds fair and ethical given we are still pre release. 

It is possible as mentioned that burning might delete very rare ones, I suggest the DAO could to keep the (say) 50 statistically rarer unclaimed jpegs out of the burning %.

**Comment by IvanGBi** [comment]
@Apestuff fully agree. Reserves must be there. But not the full 49%, wdyt?

**Comment by friouz** [comment]
I don't support giving out to those who missed the drop. The rules were clear, announced multiple times and even if you are a founder or else, you had 1 month to keep up with the news. My problem here is that it will be given only to some people, supposedly "better" than the other who missed it. Either give it to everyone who missed the drop, either don't and let it like that. I'm not comfortable with the idea to give it to some "elite founder" and not to the other who also missed.

**Comment by Genysys** [comment]
I think we should seize this opportunity and use the extra jpegs to incentivise good rich content in the lobster dao chat.

Here is my proposal:

* **People that missed out**: Give everyone that missed out 1 random NFT. tbh I think there should always have been a time decay on  how much you get and people that had been dormant for so long shouldn't retroactively get the lion's share.
* **Content Curation**: Every year , `X` amount of people (X to be decided by governance) will get `Y` (also decided by governance) amount tokens based on their contribution. The same filtering algorithm can be applied. 

I think this strengthens incentives and is aligned with long term thinking .

**Comment by DarkGhost7** [comment]
3 b is definitely a no go, imo, as @friouz said either give it to all who missed or none. The msgs in the chat were clear: claim or you won't get any. If people couldn't be bothered to claim, or follow along, then they don't deserve them now.

**Comment by IvanGBi** [comment]
Totally fair actually, you ate right. Either all or none. No subjectivity.

**Comment by 0xTR33** [comment]
Strongly agree with the latest 2 comments from @uniswalpha  and @Genysys:
1. Give everyone who missed the chance to mint 1 random LOBS independent of contribution
2. Vest the current 49% of LOBS in the DAO and release it on a monthly schedule to worthy TG users.

This worthy users should be voted monthly using Snapshot and everyone can submit his preference based on contributions. Hypothetically every user could post in a spreadsheet their monthly work (as posts or medium article shared with the chat) and let LOBS owner decide on the allocation for their work. This will probably incentivise a bit of spamming but after the first or second ban everything should be fine and will probably incentivise alpha sharing behavior from TG users.

**Comment by 0xTR33** [comment]
A monthly vote doesn't seem too much of a work in my opinion and the spreadsheet is not to be maintained by a single person, everyone can add their contribution just as a spotlight on their work (their contribution should have already been seen by everyone interested in the chat), it does not force anyone to do extra work (as you are not forced to vote) and the eventual "payout" could be arranged by simple DM to admins of whatever without the need of a bot.
Moreover, the winning contributions can be organized in a sort of "the best content in the DAO" for easy fruition and ctrl+f.
Even if this seems to be much it doesn't need to happen in the short term but burning the supply will eventually force this to possiblity to never happen.

**Comment by James0000007** [comment]
My insignificant opinion:

Open up the bot for non claimers (1 Max only sry sers) and give a +1 to all those who claimed and did not sell. This rewards those that were on a retreat/work/ family/ etc that deserve to be part of the community as they have helped build it. At the same time the +1 for all claimers who have not sold keeps the same holders list in place and rewards those that did not try flipping/selling /poor. And randomly distribute the rest like arbitary suggested amongst holders until we reach a reasonable quote for the DAO to keep ( 888 - 333 - etc) . 

That way we keep all the Art and reward those who claimed and NOT sold with a larger vote share. And we get a cool lottery that also benefits those that acquired the NFT and not minted originally. We also keep the voting share similarly structured giving a bit more weight to those with few NFTs

Distribuition will also help the DAO earn further royalties . I reckon those who claimed and not sold will hold the NFTs forever

**Comment by kirbyong** [comment]
> Randomly distribute the rest to existing lobster holders. Let RNG decides...

Of course if this was included, i'm sure existing lobster holders would be more than willing to vote this way (duh)

**Comment by storming0x** [comment]
Im biased since i was on the list that missed out, i was planned to get 2-3 lobsters based on chat contributions.

 So getting 1 at least for that in option a) would seem fair at this point.

**Comment by Jecherio1** [comment]
I suggest airdropping the rest to the inital minters. That way all the art survives and the price does not get to crazy.

I don't think opening op the contract for additional minting by people who missed is the way to go. Anyone who has been active in lobster would have not missed, anyone who just know drops in most likely is in it for the money. I also agree with @mikeinweb opening up to people who missed it opens up a can of worms from my viewpoint.

At the same point i do feel with ivan and maybe they just simply be burned.

**Comment by DaveRodman** [comment]
> Yes but this is about community :( So want to give the last open chance. To make everyone happier

I would ask the community and the dev team to please consider those of us who were idiots and made a mistake in claiming. I should have been able to claim 5 of these but due to an incredibly stupid copy/paste mistake in the Claim list my address is missing the last character so I am locked out. I’m sure that my 5 are in that list of unclaimed lobsters. I’ve got to imagine at least some other people are in the same boat. 

Was really looking forward to this. I know you have to own your own mistakes in crypto, and I do, I’m simply asking for mercy for the idiots.

**Comment by likepeas** [comment]
Those who missed had enough time. Should just keep the remainder and give it out to new people or even partnerships with projects.

**Comment by OffcierCia** [comment]
I suggest burning 1/2 of art, and then DAO will have only 20-25% of emission. In future unclaimed NFT can be used for sponsorship (for example on GitCoin).

**Comment by Jecherio1** [comment]
> 
> 
> > > Yes but this is about community :( So want to give the last open chance. To make everyone happier
> > 
> > 
> > I would ask the community and the dev team to please consider those of us who were idiots and made a mistake in claiming. I should have been able to claim 5 of these but due to an incredibly stupid copy/paste mistake in the Claim list my address is missing the last character so I am locked out. I’m sure that my 5 are in that list of unclaimed lobsters. I’ve got to imagine at least some other people are in the same boat.
> > Was really looking forward to this. I know you have to own your own mistakes in crypto, and I do, I’m simply asking for mercy for the idiots.
> 
> I should add that I believe people who made mistakes can prove they didn’t claim and show why.

And that would mean a lot of work for people involved, ignoring there would be a need for guidelines and so on. What would be fair, what would be needed for a claim to be decided to be justable. And who is the judge? unfortunately your Idea is very much specific to your situation, but not that useful to the general situation

**Comment by electroxts** [comment]
burn them all!
 unnecessary and there are no problems with a new job, people who missed it are now angry just because they saw the price 🤦‍♂️
Few

**Comment by Raclw** [comment]
My two cents, 
reg. [1] - IMHO the closest thing to what's planned would be to burn the remaining supply leaving the equivalent to 5% of the current supply in the hands of the DAO (aprox. 160 NFTs if I'm not mistaken...); at the end of the day if burned distro proportion is the same as intended (just there are less ppl /owners) 
In order to preserve the and considering it has already being produced I'll make "the unclaimed" edition, meaning, all the non claimed artworks are set in a collage PNG and generate one new token for each one of the original claimers... of that way the art does not disappear and it is a reminder of those who didn't claim (plus will be the only token which would be exactly the same to all the community members. Actually I'd make an extra one to be deposited/donated at https://muse0.xyz/ to leave it as a piece of history... Do not mean to add extrawork so if some dev likes the idea and want to help I volunteer to coordinate/help in whatever I can if needed with that issuance (I'm not a dev sadly but I believe all the infra is already there so shouldn't be that much of a task).
reg. [2] - sounds nice but start talking new distros when the first one has not concluded yet... sounds a bit like trying to modify the distro rationale... (always thought that this distro was commemorative and the nfts held are thought to be for the future, not to commemorate even further) 
reg [3] - Although I do not oppose openly... I honestly believe there has been enough time for anyone who has been barely following the chat....What is clear to me is, if at the end a second opportunity is given... it there shouldn't be no differences, no biases, all who didn't claim get their second chance (or they don't) whatever it is decided should be same to everyone. (btw I'm not including here the case of the guy who missed one number in the addy... if a separate vote is set for that particular case I'll vote yes)

**Comment by startups** [comment]
> 1] Of course, the first issue is what to do with the 49% of supply the DAO will end up with. I honestly don't know what is best here. I don't think burning all is correct, but maybe the majority of them? See twitter post by the [account](https://twitter.com/10b57e6da0/status/1446786579918467072).
> 
> 2] Also, wanna see a DAO proposal for X number of NFTs for the following: a) best recaps of solana-avalanche-fantom-polygon-xdai discussions why yes/no b) recap of the topic on regulations and how to approach token-security (non-legal advice) c) recap on malicious<>hack<>arbitrage<>theft semantic debate
> 
> 3] And an issue for those [who were late or didn't know](https://twitter.com/10b57e6da0/status/1446786579918467072). The database didn't keep records of who did or did not claim, it's an anonymized set of addresses. So you can't trust someone saying if they did or did not claim, unless we ask the bot developer to look into it. But a few founders forgot to claim, and that's sad. What could be done is the following, imho: a] ask bot developer to add more registrants and then give those people only 1 even if they had 5 before, as a fee. b] collect only founders who can be trusted and just distribute to them, without asking the bot dev to check for sybil c] ?
> 
> More incoming!

Guidelines and announcements have been sufficient for everyone to claim on time. At the end of the day, it is important what the community reflects- those who are active, actively engaging, sophisticated, and won't lose an opportunity. If users for whatever reason lost it, it's not an issue of the DAO. 

Regarding the NFTs controlled by the DAO- they should either be auctioned (a few of them monthly starting with the most premium to create awareness) and the DAO to allocate grants to create more tools for DAO infrastructure or other causes that the DAO will investigate. They could even be used to invest collectively in new projects that arise from the community.

In any case, distributing to those who missed the deadline doesn't make sense.

**Comment by Apestuff** [comment]
> @Apestuff fully agree. Reserves must be there. But not the full 49%, wdyt?



> Strongly agree with the latest 2 comments from @uniswalpha and @Genysys:
> 
> 1. Give everyone who missed the chance to mint 1 random LOBS independent of contribution
> 2. Vest the current 49% of LOBS in the DAO and release it on a monthly schedule to worthy TG users.
> 
> This worthy users should be voted monthly using Snapshot and everyone can submit his preference based on contributions. Hypothetically every user could post in a spreadsheet their monthly work (as posts or medium article shared with the chat) and let LOBS owner decide on the allocation for their work. This will probably incentivise a bit of spamming but after the first or second ban everything should be fine and will probably incentivise alpha sharing behavior from TG users.

I don't agree with this. this will just create more "airdrop farmers" and create unauthentic behaviour.

**Comment by Apestuff** [comment]
> @Apestuff fully agree. Reserves must be there. But not the full 49%, wdyt?

Personally I'd like to see the DAO keep at least 10% of the total pieces. 10% isn't a whole lot and it gives more flexibility in future for plans. Ideally I'd like for 2.5% of the pieces to be earmarked for heritage and charity purposes and the other 2.5% for the community badge of honour.   All in all I think its only fair for the community (read: people who actually responded and claimed) to decide.

**Comment by ncerovac** [comment]
LobsterDAO is an amazing community that gathers a lot of pure heart people that want to see our decentralized world excel. **I'm 100% against burn**, I don't see the reason to burn any NFTs. Just during last year, we had an influx of great people that are contributing to the community and I think in the next 5 years we will see even more. 

With that in mind, I'm proposing few things where we can allocate ~40% out of 49%

1. **allocate a portion to research rewards** - CIA can be in charge of the flow there. we list topics people apply make research, few gib feedback, researcher fix it, we reward and publish.

2. **allocate a portion to projects like gitcoin, ethernaut, security researchers etc.** - like for the next 10 rounds we can auction Lobs NFTs and fund amazing teams and individuals.

3. **allocate portion for members contribution in next 3 years** 

there are already few individuals that would be more than happy to contribute their time to see this works out.

**Comment by tsarpaul** [comment]
I voted no.
Not sure what's going to be the point of the DAO if it has no war chest to execute on.
Burning is effectively stripping the lobsters to a plain NFT collection, and that wouldn't be much fun.
We can choose to do that at a later date.

**Comment by lobsterdao** [comment]
> <img alt="Screenshot 2021-10-09 at 13 48 11" width="783" src="https://user-images.githubusercontent.com/40795256/136656702-3a911347-c361-4013-b06f-e57a84ba75cc.png">
> 
> Ah and one more - this was my bad! The user didn't have tooooo many messages, but was valuable. I believe the number this would qualify for is 1. The DAO should gib the 1 piece, this is fairly trivial. Screenshot as proof.

Gave from my bag is oke: https://etherscan.io/tx/0xd9176c7d765f9130ab1d9eba15081468e170d6cdbd8041168c5010671621ded1

**Comment by tartakovsky** [comment]
Guys, check out the actual [LIP1 issue](https://github.com/lobster-dao/overview/issues/13).

Read what Ivan described. As far as anybody can see so far, there is nothing actionable to vote for :)

Let's move the discussion there.

**Comment by IvanGBi** [comment]
> <img alt="Screenshot 2021-10-09 at 13 48 11" width="783" src="https://user-images.githubusercontent.com/40795256/136656702-3a911347-c361-4013-b06f-e57a84ba75cc.png">
> 
> Ah and one more - this was my bad! The user didn't have tooooo many messages, but was valuable. I believe the number this would qualify for is 1. The DAO should gib the 1 piece, this is fairly trivial. Screenshot as proof.

Quoting: sent from person allo, is fine.

https://etherscan.io/tx/0xd9176c7d765f9130ab1d9eba15081468e170d6cdbd8041168c5010671621ded1

**Comment by gogolitalia** [comment]
I would do a random lottery….

**Comment by RSivakov** [comment]
Sorry for not very good English.

**DISTRIBUTION:**

Consider vesting.
See:
- There are worthy people and their number is increasing (they fall into the Merkle Tree and supplement it over time). In total, there is a list. He grows. Can't find addresses. Knowing the address, you can check its presence in the list.
- There is a gradual release of NTF (the number is assigned at the beginning of the period and allows you to make a mint for people from the list in the first paragraph. The number does not carry over to the next period if there was no requirement.)
- No notifications (or are they needed?). There are no promises. There are no guarantees, speculations and so on. There is a growing list and portions of "occasionally" mint available.

It turns out that we have a gradual fair distribution. Distribution occurs at the intersection of sets: worthy * willing * a little bit of luck.

(I think there are three different statuses to be specified:
-- Already received.
-- Not on the list
- Is on the list, but not available at the moment. Try making a mint in the future.

People themselves will check the possibility of mints from time to time.

The dissatisfied will not have the opportunity to blame someone from other people, tk. this process provides a fully automatic mechanism.

Those who want to buy the opportunity for mints will have many difficulties, again, due to the automation.

And also, we will not lose the entourage, pathos, sparkles, mystery ... heh.

Thus, the gradual increase in the number of participants will be uniform, fair and, in my opinion, quite nice.


**CONTINUATION AND DEVELOPMENT**


Further, along the way, it is possible to issue additional NFTs to existing participants personally for some merits (these can be personal ones with the name of NFT, or NFTs that cannot vote and donated by the community to a person in order to make a valuable gift that can be turned into money ...)

You can also give NTF to projects. Contracts.

You can give non-movable nft.


**Losing your mind a little**


You can also add a certain amount of NTF into an additional DAO and issue tokens equally to everyone. The tokens will be backed by collector demand for NFT (as we can see now at opensea). This will give you an interesting dao-stable coin. Heh.
